ITR/interfaces.py

```diff
@@ -457,17 +456,24 @@ class PortfolioCompany(BaseModel):
     investment_value: float
     user_fields: Optional[dict]
 
-
 def pint_ify(x, units='dimensionless'):
-    if 'units' in units:
-        units = units['units']
-    if x is None or unp.isnan(x):
+    global stop_pint_ify
+
+    try:
+        if 'units' in units:
+            breakoint()
+            units = units['units']
+    except TypeError:
+        pass
```

Comment:
it's a `TypeError` - is `pass` a safe move?
what's the value of `units` variable, if we got this exception here?

Reply:
The function `pint_ify` is the last vestiage of the bad Pydantic code I started writing last year.  I think there are four places left that call it, and a next step is to look carefully at those cases, rewrite/check that it's not needed, then remove it.  Good catch.

Reply:
Last vestiges removed...